webrtc-stats.html

```diff
@@ -430,8 +430,8 @@
             <dl data-link-for="RTCRTPStreamStats" data-dfn-for="RTCRTPStreamStats" class=
             "dictionary-members">
               <dt>
-                <dfn><code>ssrc</code></dfn> of type <span class=
-                "idlMemberType"><a>DOMString</a></span>
+                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned
+                long</a></span>
               </dt>
               <dd></dd>
```

Comment:
While you're at it, can you fill in the explanation (the part between <dd> and </dd>) with "32-bit integer value per [[RFCxxxx]]"?